### ISSUE TITLE
A couple of questions

### ISSUE THREAD

**Issue by bropines** [dmMaze/BallonsTranslator#18]
The program when performs all translations, painting over even what is not needed. Like I don't want any cloud to be painted over and translated, with text, but it can't be undone. There is no paint layer as such. I would like to be able to remove probably painted and translated pieces

If you don't understand anything, I can attach a video)

**Comment by dmMaze** [comment]
Did you mean manually perform/correct text removal(i.e., inpainting)?
Actually there is a inpainted layer, it just doesn't show explicitly in the ui. And there is a mask layer to show which part are inpainted.
To 'erase' unwanted inpainted results, use the inpainted tool or rect tool with your right button pressed, please refer to the following gifs and notice the keystrokes are shown on the top left corner:

https://github.com/dmMaze/BallonsTranslator/blob/master/doc/src/imgedit_inpaint.gif
https://github.com/dmMaze/BallonsTranslator/blob/master/doc/src/rect_tool.gif

**Comment by bropines** [comment]
It worked, thanks
Does the project have its own discord/telegram to throw off bugs and error reports?
Github allows it, but often bugs are simply associated with the departure of python modules.

**Comment by dmMaze** [comment]
> Does the project have its own discord/telegram to throw off bugs and error reports?

Nope.


> Github allows it, but often bugs are simply associated with the departure of python modules

I'll add some instructions to readme for running the source code.

**Comment by bropines** [comment]
One more question before closing the issue. Can you add a config to translate to another language? I would like to translate the program settings into my language, but I don't really want to get into the code.

**Comment by dmMaze** [comment]
This program can be translated using linguist.exe, which should be installed with pyqt.
<img width="521" alt="image" src="https://user-images.githubusercontent.com/51270320/184656579-2fee5a5a-23f3-46ac-9b7f-bb63a0f73461.png">
If you are comfortable with it, edit ```ballontranslator/data/translate/template.ts``` in linguist and export to ```ballontranslator/data/translate/[LANGUAGE].qm```

You can print ```QLocale.system().name()``` in the main function of ```ballontranslator/__main__.py``` to determine the [LANGUAGE] of your system:
https://github.com/dmMaze/BallonsTranslator/blob/d424cf7dbaaa884d7ba9869528053bd1c6d5a5b2/ballontranslator/__main__.py#L34  

Please notice the template.ts is a bit out of date, I'll update it after I merge the ```textlayout``` branch, perhaps later this week.

**Comment by dmMaze** [comment]
> edit `ballontranslator/data/translate/template.ts`

Update: run ```python ballontranslator\scripts\update_translation.py``` to generate or update [LANGUAGE].ts

**Comment by bropines** [comment]
A couple more questions.
1. Is it possible to add support for yandex translate?
2. Is it possible to translate individual clouds as in this project (https://github.com/AntonRls/Manga-Translator)?

**Comment by dmMaze** [comment]
> A couple more questions.

1. I don't have an access key required by yandex, however, if you're familar with python and know how to use yandex's api, it would be easy to add it to the program, you can refer to [this pr](https://github.com/dmMaze/BallonsTranslator/pull/3)  
2. You mean ocr&translate single textblock created by user? No, I'll consider adding it to the right-click menu in the future. By the way, all translated texts are interactly editable.  
3. try ```pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu116```, AMD GPU would not work on windows.

**Comment by bropines** [comment]
![image](https://user-images.githubusercontent.com/57861007/186140821-ddb68d58-df52-4409-9314-c6f27312e1a0.png)
When you click once on a spot with the text-coloring tool, it gives an error. Just right-click on the spot

**Comment by bropines** [comment]
And we'll finish it. I accidentally deleted a block. 
1. I did not understand how to make mine with text.
2. Is it possible to apply OCR and all operations to only one page?

**Comment by bropines** [comment]
And I still don't have cuda support. I have two video cards. One is built-in, the second is discrete. Maybe because of this, cuda does not work?
![image](https://user-images.githubusercontent.com/57861007/186149099-16149855-e5b9-4d69-9591-ae1045387575.png)

**Comment by Snowad14** [comment]
Use your nvidia card, go to your terminal, type "python" then "import torch" and "torch.cuda.is_available()" and make sure it returns True otherwise your installation of pytorch is not well installed or you do not have install cuda on your system " nvcc --version" to see if you have installed cuda well

**Comment by dmMaze** [comment]
> When you click once on a spot with the text-coloring tool, it gives an error. Just right-click on the spot

fixed d55a1a054b2734dac46e9af8c9bdb138a467143f


> 1. I did not understand how to make mine with text

Activate text editing mode and the **rectangle button** next to it:
<img width="131" alt="image" src="https://user-images.githubusercontent.com/51270320/186154412-70a00b1d-aeec-4e82-9bdb-4fd78347ae72.png">

then drag mouse on the canvas with right button pressed, see following gif, it's after delete operation
<img src="https://raw.githubusercontent.com/dmMaze/BallonsTranslator/master/doc/src/textedit.gif">




> 2\. Is it possible to apply OCR and all operations to only one page?

No

**Comment by bropines** [comment]
Now I think my friend and I will make a couple of variants of the logo (icon). I'll send the options here.
